public/static/docs/api-reference/index.md

```diff
@@ -0,0 +1,30 @@
+# Python API
+
+When you [install](/doc/install) DVC with an package manager like `pip` or
+`conda`, the `dvc` package becomes available to the corresponding Python
+environment. While most of the package implements our
+[command-line tool](/doc/command-reference), we wrote the `dvc.api` module to
+expose special functions you can use in your Python source code.
+
+> For API use, we **strongly** recommend having `dvc` in a requirements file for
+> your Python project, and installing it with a package manager (in a
+> [virtual environment](https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments)
+> preferably).
+
+To use the API, import the module first with:
+
+```py
+import dvc.api
+```
+
+This reference provides the details about the API functions (inside `dvc.api`):
```

Comment:
probably should come first on this page. Just the first section, not need to include the second at all. We need also to briefly describe the essence of the API - providing access from Python to the data stored under DVC control.

Reply:
> probably should come first on this page 

Addressed in https://github.com/iterative/dvc.org/pull/908#pullrequestreview-348449124 above.

> We need also to briefly describe the essence of the API

Done, please check.